models/index.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var config = require('../lib/config');
+
+var fs = require('fs');
+var path = require('path');
+var Sequelize = require('sequelize');
+var basename = path.basename(module.filename);
+var sequelize = new Sequelize(config('databaseURL'), {
+    logging: process.env.NODE_ENV && process.env.NODE_ENV !== 'test'
+});
+
+var db = {};
+
+fs
+    .readdirSync(__dirname)
+    .filter(function(file) {
+        return (file.indexOf('.') !== 0) && (file !== basename);
+    })
+    .forEach(function(file) {
+        if (file.slice(-3) !== '.js') return;
+        var model = sequelize['import'](path.join(__dirname, file));
+        db[model.name] = model;
+    });
+
+Object.keys(db).forEach(function(modelName) {
+    if (db[modelName].associate) {
+        db[modelName].associate(db);
+    }
+});
+
+db.sequelize = sequelize;
+db.Sequelize = Sequelize;
+
+module.exports = db;
```

Comment:
Definitely. That's the problem with Singletons.

This particular file is auto-generated by `sequelize init`. I tend to follow suggested patterns until we run into problems, so that we can keep the maintenance and cognitive costs low, but I definitely want to keep thinking about this.